packages/web/pages/index.js

```diff
@@ -20,16 +20,13 @@ const Home = ({ emailConfirmation }) => {
 	return (
 		<>
 			<Hero />
-			<TechnologiesSection
-				header={t('common:featuredSolutions')}
-				technologies={fullTechnologies}
-				bgColor={colors.gray98}
-			/>
-			<TechnologiesSection
-				header={t('common:recentSolutions')}
-				technologies={technologies}
-				bgColor={colors.whiteSmoke}
-			/>
+			{!!technologies?.length && (
```

Comment:
Muit bom! Sempre bom manter essas verificações até porque não sabemos como será no início da plataforma :100: